nanoDFT/experimental_pmap_nanoDFT.py

```diff
@@ -2,14 +2,8 @@
 import jax
 import jax.numpy as jnp
 import numpy as np
-import os
-os.environ['OMP_NUM_THREADS'] = "16"
```

Comment:
This was used when running multiple IPUs in parallel by starting different python processes. So we definitely want to remove here, nice catch!